### DISCUSSION TITLE
Feature: Headless Draggable components

### DISCUSSION THREAD

**Discussion by lotestudio** [tailwindlabs/headlessui#665] [Ideas]
Idea for headless Draggable components using html5 draggable API. Something like
 https://web.dev/drag-and-drop/?gclid=CjwKCAjwzruGBhBAEiwAUqMR8BCme1WoWC3FaqNYveFKSptroR3AEND-yR3Uv6nxKdiSUoNTAsWG8hoC3ywQAvD_BwE

**Comment by LoganDark** [comment]
[`react-draggable`](https://github.com/react-grid-layout/react-draggable) contains both stateful and controlled draggable components, and can definitely be used to implement a nicer, richer drag than the HTML5 API. Are you sure this does not fit your use case?
